packages/build-info/src/file-system.test.ts

```diff
@@ -85,14 +86,28 @@ describe.concurrent('Test the platform independent base functionality', () => {
   })
 
   test('join should join path segments and always replace to posix style', ({ fs }) => {
+    expect(fs.join('/', 'package.json')).toBe('/package.json')
+    expect(posixJoin('/', 'package.json')).toBe('/package.json')
+    expect(fs.join('', 'package.json')).toBe('package.json')
+    expect(posixJoin('', 'package.json')).toBe('package.json')
+    expect(fs.join('', '', '', 'package.json')).toBe('package.json')
+    expect(posixJoin('', '', '', 'package.json')).toBe('package.json')
```

Comment:
it looks like we're asserting that `fs.join` behaves just like `posixJoin`. Maybe we could refactor the test to make that more obvious? The way it looks right now, it's hard for me to spot the differences and see what the asserted behaviour is between them.

Reply:
so you would prefer a loop that tests both?

Reply:
yeah, something like a set of test-cases that's iterated over

Reply:
Addressed in https://github.com/netlify/build/pull/5042/commits/91472cc64f031a64510a96b3741e5b74a02f2d19